.github/PULL_REQUEST_TEMPLATE/docs_pr.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Documentation improvement
+about: Changes to things other than code
+title:
+labels: documentation, needs triage
+assignees: @Grossley
+
+---
+
+## Description
+<!-- A clear, in-depth description of what the changes are. Reference existing issues and add screenshots if necessary! -->
+
+### Notes
+<!-- Any notes or closing words -->
```

Comment:
While GitHub technically does support multiple PR templates, it does not do so in an intuitive way. Via the web interface, it will only ever open the first (?) PR template, if one wants to use a different PR template, then one needs to open a specific direct link which opens GitHub in a PR window with the specific template applied. 
I'd personally just get rid of this template, and make the one below more generalized in the headers.